genie-client/src/main/java/com/netflix/genie/client/JobClient.java

```diff
@@ -191,7 +191,9 @@ public void writeTo(final BufferedSink sink) throws IOException {
      * @throws IOException          For Network and other IO issues.
      */
     public List<JobSearchResult> getJobs() throws IOException, GenieClientException {
-        return this.getJobs(null, null, null, null, null, null, null, null, null, null, null, null, null, null, null);
+        return this.getJobs(null, null, null, null, null, null,
+            null, null, null, null, null, null,
```

Comment:
Can you do one parameter per line (repository style convention, I'm surprised checkstyle is ok with this.

Reply:
It complained about line length, so I split this way. Will make it one parameter per line.